src/contexts/Transaction/TxHistoryProvider.tsx

```diff
@@ -22,20 +22,27 @@ const pollTransaction = async (
   if (!txLog.tx?.hash || txLog.status !== TxStatus.pending) {
     return txLog
   }
-
   const response = await readProvider.getTransaction(txLog.tx.hash)
-  // if no response, then the tx is cancelled.
+  const oneMinuteAgo = nowSeconds() - 1 * 60
+  // if no response and tx was created over a minute ago, then the tx is cancelled.
   if (!response) {
```

Comment:
I think the problem that this was getting called too quickly. It would give a null response instantly, tx would get marked as failed, and not be checked again. So I added a delay here, so tx doesn't get marked as failed unless the null response occurs 1min after the tx was created